src/Extensions/Abstractions/Activities/ActivityExtensions.cs

```diff
@@ -37,6 +37,24 @@ public static class ActivityExtensions
 				HealthCheckMarkerValue,
 				StringComparison.OrdinalIgnoreCase);
 
+		/// <summary>
+		/// Returns true if activity is marked as Performance test
+		/// </summary>
+		/// <param name="activity"></param>
+		/// <returns></returns>
+		public static bool IsPerformanceTest(this Activity activity) =>
```

Comment:
How are you planning to add this flag? Maybe we need the corresponding method `MarkAsPerformanceTest` and it's done for the health check marker?

Also could you please add some explanation of why this marker is needed and how it would be used?

Reply:
Yes, please make sure anything going into public-facing repos is documented to the standard you'd expect if you were referencing an external library.

Reply:
This flag will be added by specifying the value "PerformanceMarker=0" or "PerformanceMarker=500" for the Header Correlation-Context.  I don't think we need a method 'MarkAsPerformanceTest' as the header value will be added manually when running performance tests.

This header value will be specified when running performance tests which will by-pass calls to partner services.

Reply:
@K-Cully  Is there a location where this should be documented?  Or should documentation be added to the XML tag for the method?

Reply:
XML for methods would the best place

Reply:
Let's join discussions here https://github.com/microsoft/Omex/pull/386#discussion_r691145343